python-packages/order_utils/src/zero_ex/order_utils/__init__.py

```diff
@@ -9,3 +9,145 @@
 --networkId 50 -m "concert load couple harbor equip island argue ramp clarify
 fence smart topic"``.
 """
+
+import json
+from typing import Dict
+from pkg_resources import resource_string
+
+from mypy_extensions import TypedDict
+
+from eth_utils import is_address, keccak, to_checksum_address, to_bytes
+from web3 import Web3
+from web3.utils import datatypes
+import web3.exceptions
+
+
+EXCHANGE_ABI = json.loads(
+    resource_string("zero_ex.contract_artifacts", "artifacts/Exchange.json")
+)["compilerOutput"]["abi"]
```

Comment:
Do we need to load this in every file that needs it? Or can we load it somewhere once?